### PR TITLE
kernel 3.19 images and overlay fs.

### PR DESCRIPTION
**Pull request by enxebre**
Setting the base images with kernel 3.19
Setting overlay as fs.
Paving the way for overlay network.

### CODE REVIEW

File: roles/docker/tasks/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     dest: /etc/default/docker
     state: present
     regexp: ^DOCKER_OPTS=.*--graph.*
-    line: 'DOCKER_OPTS=\"$DOCKER_OPTS --graph={{ docker_graph_dir }} --dns 172.17.0.1 --dns 8.8.8.8 --dns-search service.{{ consul_domain }} \"'
+    line: 'DOCKER_OPTS=\"$DOCKER_OPTS --storage-driver=overlay --graph={{ docker_graph_dir }} --dns 172.17.0.1 --dns 8.8.8.8 --dns-search service.{{ consul_domain }} \"'
```
Comment (wallies) [inline roles/docker/tasks/main.yml]: do you want to do --storage-driver={{ docker_storage_driver }} instead and then in defaults put overlay

**Comment by pmbauer** [comment]
I may have missed it, but is there a corresponding set of packer commits to upgrade the base images to 3.19?

File: roles/docker/tasks/main.yml (hunk 1 above)
Comment (enxebre) [inline roles/docker/tasks/main.yml]: Make sense I'll include that as part of a different PR

**Comment by enxebre** [comment]
@pmbauer we are using now http://cloud-images.ubuntu.com/trusty/current/hwe-v/

**Comment by pmbauer** [comment]
@enxebre what I mean is, I don't see any changes here https://github.com/Capgemini/Apollo/commits/master/packer

**Comment by enxebre** [comment]
yeah there is no change in packer because the new base amis (upated in variables.tf) include the kernel 3.19 already :) http://cloud-images.ubuntu.com/trusty/current/hwe-v/trusty-server-hwe-v-cloudimg-amd64-kernel-info.txt

**Comment by pmbauer** [comment]
Ah; sorry; I didn't realize 3.19 was coming in 14.04 LTE. Thanks.

**Pull request merged**